packages/git/src/browser/blame/blame-contribution.ts

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { inject, injectable } from 'inversify';
+import { FrontendApplicationContribution, FrontendApplication, KeybindingContribution, KeybindingRegistry } from "@theia/core/lib/browser";
+import { CommandContribution, CommandRegistry, Command, MenuContribution, MenuModelRegistry, Disposable, DisposableCollection } from '@theia/core/lib/common';
+import { BlameDecorator } from './blame-decorator';
+import { EDITOR_CONTEXT_MENU, EditorManager, EditorKeybindingContexts, EditorWidget } from '@theia/editor/lib/browser';
+import { BlameManager } from './blame-manager';
+import URI from '@theia/core/lib/common/uri';
+import { DiffUris } from '@theia/editor/lib/browser/diff-uris';
+
+export namespace BlameCommands {
+    export const SHOW_GIT_ANNOTATIONS: Command = {
+        id: 'git.editor.show.annotations',
+        label: 'Show Git Annotations'
```

Comment:
to discuss: Should not it be `Git: Show Blame Annotations`? I was looking for `blame` in the quick open.

Reply:
I recall a discussion about that, some tools switched form `blame` to `annotation`, because it would sound like less finger pointing.

Reply:
In VS Code is `Show Blame Annotations`, in GitHub is just `Blame`. I concerned about the discoverability of a command.

Also thinking about it, the label is not aligned with https://github.com/theia-ide/theia/issues/1119.

Reply:
I think Eclipse, IntelliJ, and VS call it just `Annotations`.

Reply:
You're right about #1119.

Reply:
`Git: Show Blame Annotations` or just `Git: Show Annotations` ?

Reply:
done.